packages/teuchos/comm/src/Teuchos_StackedTimer.hpp

```diff
@@ -589,13 +589,14 @@ class StackedTimer
    */
   struct OutputOptions {
     OutputOptions() : output_fraction(false), output_total_updates(false), output_histogram(false),
-                      output_minmax(false), num_histogram(10), max_levels(INT_MAX),
+                      output_minmax(false), output_procmax(false), num_histogram(10), max_levels(INT_MAX),
                       print_warnings(true), align_columns(false), print_names_before_values(true),
                       drop_time(-1.0) {}
     bool output_fraction;
     bool output_total_updates;
     bool output_histogram;
     bool output_minmax;
+    bool output_procmax;
```

Comment:
Prefer C++11 member initializers.  Then you could just write `OutputOptions() = default;` (or not write anything at all).